datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -479,11 +511,23 @@ msgstr "Ungültige RAM-Grösse"
 msgid "Invalid storage size"
 msgstr "Ungültige Speicher-Grösse"
 
-msgid "is not a proper name"
-msgstr "ist kein gültiger Name"
+msgid "Another user exists with that email!"
+msgstr ""
```

Comment:
"Ein Benutzer mit dieser E-Mail existiert bereits."

Reply:
That text does not exist now.